packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -23,6 +24,26 @@ const int defaultSplitterWidth = 10;
 const String runInProfileModeDocsUrl =
     'https://flutter.dev/docs/testing/ui-performance#run-in-profile-mode';
 
+// GA dimensions:
+String userAppType = ''; // dimension1
+String userBuildType = ''; // dimension2
+String userPlatformType = ''; // dimension3
+
+void computeApplicationState(
+  bool isAnyFlutterApp,
```

Comment:
make these @required optional named parameters as otherwise it would be really easy to put the booleans in the wrong order.

Reply:
Eliminated parameters.